Documentation/guides/building-apps/build-items.md

```diff
@@ -38,6 +38,24 @@ Used to provide an AOT profile, for use with profile-guided AOT.
 It can be also used from Visual Studio by setting the `AndroidAotProfile`
 build action to a file containing an AOT profile.
 
+## AndroidAppBundleMetaDataFile
+
+Specifies a file that will be included as metadata in the Android App Bundle.
+The format of the flag value is '<bundle-path>:<physical-file>' where
```

Comment:
This should use backticks `` ` `` not `'`, as Markdown "passes through" `<looks-like-an-html-tag-to-me>`.  If you view the Markdown preview for this page, you see:

>  The format of the flag value is ':' where 'bundle-path' denotes…

which is *not* what you intended. :-)

We may also want to link to the appropriate `bundletool build-bundle` documentation: https://developer.android.com/studio/build/building-cmdline

Reply:
This doc section should also mention that when `$(AndroidProguardMappingFile)` isn't empty and `$(AndroidCreateProguardMappingFile)`=True (w/ appropriate doc links), we automatically create an item in this item group.